.github/workflows/publish_docs_to_wiki.yml

```diff
@@ -13,6 +13,16 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 30
     steps:
+      - name: Cancel previous runs
+        uses: styfle/cancel-workflow-action@0.8.0
+        with:
+          access_token: ${{ github.token }}
+
+      - name: Checkout the repository
+        uses: actions/checkout@v2.3.2
```

Comment:
Same here: `actions/checkout@v3`

Reply:
Updated in [f0826b7](https://github.com/nimblehq/android-templates/pull/456/commits/f0826b7946679981358e529de63faa6c39bcc8b9)